monero-outreach-docs/translations/nl/PSA/files/dontshare.md

```diff
@@ -0,0 +1,11 @@
+# Deel nooit je privésleutels
+
+Zijn je privésleutels en hersteltekst veilig, dan is je Monero dat ook. Want ze zijn de enige manier waarop je je wallet kan openen en je XMR kan zenden/ontvangen. Iedereen met wie je privésleutels of hersteltekst deelt, of met technische expertise ze van je te stelen, heeft evenveel toegang to je wallet als jijzelf.
+
+Misschien denk je dat het een goed idee is je privésleutels gewoon elders te bewaren. Maar wat als grootmoeder je instructies op de vensterbank laat liggen en een voorbijganger het noteert? Dan word je op een goede dag wakker met al je XMR verzonden naar een onbekend adres. Met Monero's privacy kan niet achterhalen wiens onbekende wallet het betreft, en zal je je XMR nooit meer terugkrijgen.
+
+Het kan ook voorkomen dat Monero's code gevorkt wordt tot een nieuw cryptoproject met de claim: 'vertienvoudig je bedrag in ruil voor je privésleutels'. Kom je dit tegen, blijf dan uit de buurt! Niet alleen probeert men je wellicht op te lichten, mogelijk wordt de hele integriteit van het Monero-netwerk ondermijnt. Oplichters hangen af van je hebzucht om hun schade te berokkeren.
+
+### Val niet ten prooi
+
+Het enige attribuut van de wallet dat je wil delen is je leessleutel - en zelfs dan ook enkel om inkomende transacties aan te tonen. Is dit niet nodig, deel dan geen sleutels. Als er meer wordt gevraagd dan je publieke walletadres (of soms je leessleutel), dan probeert men je waarschijnlijk op te lichten en te bestelen. Kortweg: deel nooit je privésleutels.
```

Comment:
'Leessleutel' is a term we haven't been using so far, but we should discuss it. It's simpler and more natural than 'alleen-lezen sleutel'.